frontend/src/pages/questions/ViewQuestion.tsx

```diff
@@ -46,31 +47,39 @@ const ViewQuestion: React.FC = () => {
   }
 
   return (
-    <Box width="100%" height="85vh" my={5}>
+    <Box width="100%" height="100vh" my={5}>
       <Allotment>
         <Allotment.Pane>
-          <VStack as="div" style={{ overflowY: 'auto', height: '100%', padding: '16px' }}>
-            <Heading as="h1" size="xl" textAlign="center">
-              {question.title}
-            </Heading>
-            <Text fontSize="md" color={colourScheme} mt={2}>
-              <span style={{ fontWeight: 'bold' }}>Complexity: </span>
-              <QuestionComplexityTag questionComplexity={question.complexity} />
-            </Text>
-            <Text fontSize="md" color={colourScheme} mt={2}>
-              <span style={{ fontWeight: 'bold' }}>Categories:</span> {question.categories.join(', ')}
-            </Text>
-            <Text fontSize="md" color={colourScheme} mt={2}>
-              <span style={{ fontWeight: 'bold' }}>Link to Question: </span>
-              <Link href={question.linkToQuestion}>{question.linkToQuestion}</Link>
-            </Text>
-            <Divider mt={4} />
-            <VStack align="start" spacing={4} mt={4}>
-              <Heading as="h2" size="md">
-                Description
+          <VStack as="div" alignItems="start" style={{ overflowY: 'auto', height: '100%', padding: '16px' }}>
+            <VStack alignSelf="center" alignItems="center">
+              <Heading as="h1" size="xl" textAlign="center">
+                {question.title}
               </Heading>
-              <Text whiteSpace="pre-line">{question.questionDescription}</Text>
```

Comment:
Hm, should this `whiteSpace="pre-line"` be added back? Seems to be running into issues rendering the "\n" characters right now without this property 🤔

Reply:
yeap i removed by accident

Reply:
thanks :)